docs/operator-docs/cmr-operations.md

```diff
@@ -21,14 +25,17 @@ $ curl --request PUT https://example.com/granules/MOD11A1.A2017137.h19v16.006.20
 --data '{ "action": "applyWorkflow", "workflow": "PublishGranule" }'
 ```
 
-This will take an unpublished granule and publish it to CMR.
+Regardless of the method of request submission, this will take an unpublished granule and publish it to CMR.
```

Comment:
```suggestion
```

Reply:
To clarify, your suggestion is this line is unnecessary?

Reply:
Ah, combined with the change further up. Fair enough. 👍